visitz/Visitz/Views/Entity/Navigation/EntityNavViewModel.cs

```diff
@@ -135,14 +144,29 @@ private async Task SetupDraftsObserver()
 		}
 	}
 
-	private void RealmQueryMap_ItemsChanged(object sender, (Type Type, IRealmCollection<IRealmObject> Items, ChangeSet Changes) e)
+	private async void RealmQueryMap_ItemsChanged(object sender, (Type Type, IRealmCollection<IRealmObject> Items, ChangeSet Changes) e)
 	{
 		if (e.Type == typeof(NoteDraft))
 			Notes.HasDraft = e.Items.Any();
 		else if (e.Type == typeof(AssessmentDraft))
 			SafetyAssessment.HasDraft = e.Items.Any();
 		else if (e.Type == typeof(AttachmentDraft))
 			Attachments.HasDraft = e.Items.Any();
+		else if ((e.Type == typeof(CaseloadItem)) && (e.Changes?.DeletedIndices?.Length > 0))
```

Comment:
Can remove extra parentheses:

```suggestion
		else if (e.Type == typeof(CaseloadItem) && e.Changes?.DeletedIndices?.Length > 0)
```